job-service-container/src/test/java/com/hpe/caf/services/job/api/JobServiceFilterIT.java

```diff
@@ -44,6 +44,8 @@
  */
 public class JobServiceFilterIT
 {
+    private static final String RABBIT_PROP_QUEUE_TYPE = "x-queue-type";
```

Comment:
Move these private static finals to 
JobServiceConnectionUtil
and make them public